testament/categories.nim

```diff
@@ -501,37 +480,36 @@ proc testNimblePackages(r: var TResults; cat: Category; packageFilter: string, p
   let packagesDir = "pkgstemp"
   var errors = 0
   try:
-    for name, url, cmd, hasDep, useHead in listPackages(part):
+    let importantList =
```

Comment:
PR title "Use cligens release version" needs to be updated to reflect those bigger changes